### ISSUE TITLE
Remove yml_to_req.py

### ISSUE THREAD

**Issue by karkir0003** [DSGT-DLP/Deep-Learning-Playground#868]
**Problem:** DLP is moving to using `poetry` as our package manager for our Python backend. This means that we won't be using conda + requirements.txt setup. 

**Solution:** Because we removed the `conda/environment.yml` file in the [Poetry Migration PR](https://github.com/DSGT-DLP/Deep-Learning-Playground/pull/863), we don't need the `yml_to_req.py` file since `poetry` has a built in command to convert `pyproject.toml` to `requirements.txt` should that need come up. 

TLDR: Pretty much remove `yml_to_req.py` and ensure that the unit tests pass.

**Comment by farisdurrani** [comment]
Solved in #947
